core/platform/models.py

```diff
@@ -276,18 +276,28 @@ def import_app_identity_services(cls) -> ModuleType:
 
     @classmethod
     def import_azure_speech_synthesis_services(cls) -> ModuleType:
-        """Imports and returns azure_speech_synthesis_services module.
```

Comment:
This is actually weird btw. The stuff in platform/models is supposed to be generic and not tied to a specific provider. Basically, this should be import_speech_synthesis_services and one impl of that would be dev_mode_speech_synthesis_services and the other would be prod_speech_synthesis_services. The former returns dummy files and the latter looks at the language/accent, chooses the provider based on that, sends the string to a provider-specific impl, gets the voiceover, and returns it in a standard format that should be common across all external providers.

Having "azure" here means that code above this layer knows about azure and will have special-cases relating to it, and that's not really desirable.

Reply:
Yes @seanlip, this makes sense. I have updated the code.